crates/derive-alloy/src/pipeline.rs

```diff
@@ -1,14 +1,6 @@
 //! Helper to construct a [DerivationPipeline] using online types.
 
-use kona_derive::{
-    attributes::StatefulAttributesBuilder,
-    pipeline::{DerivationPipeline, PipelineBuilder},
-    sources::EthereumDataSource,
-    stages::{
-        AttributesQueue, BatchProvider, BatchStream, ChannelProvider, ChannelReader, FrameQueue,
-        L1Retrieval, L1Traversal,
-    },
-};
+use kona_derive::prelude::*;
```

Comment:
Not a huge fan of importing `*`, it causes a lot more things to be linked with this module, making resulting binaries a bit bigger. Makes it a good bit harder for LLVM to do dead code elimination.